src/piral-build.test.ts

```diff
@@ -7,63 +7,192 @@ runTests('piral-build', ({ test, setup }) => {
     await ctx.run(`npm i emojis-list@3.0.0`);
   });
 
+  test('release-standard-template', 'can produce a release build', ['build.piral'], async (ctx) => {
+    await ctx.run(`npx piral build --type release`);
+
+    await ctx.assertFiles({
+      'dist/release/index.html'(content: string) {
+        expect(content).not.toBe('');
+      },
+    });
+  });
+
+  test('emulator-standard-template', 'can produce an emulator build', ['build.piral'], async (ctx) => {
+    await ctx.setFiles({
+      'package.json'(content: string) {
+        const packageJson = JSON.parse(content);
+        packageJson.name = 'app-shell';
+        return JSON.stringify(packageJson, undefined, 2);
+      },
+    });
+
+    await ctx.run(`npx piral build --type emulator`);
+
+    await ctx.assertFiles({
+      'dist/emulator/*.tgz'(files: Array<string>) {
+        expect(files).toHaveLength(1);
+        ctx.setRef('emulator', files[0]);
+      },
+    });
+  });
+
+  test('emulator-sources-standard-template', 'can produce emulator sources build', ['build.piral'], async (ctx) => {
+    await ctx.run(`npx piral build --type emulator-sources`);
+
+    await ctx.assertFiles({
+      'package-lock.json': true,
```

Comment:
package-lock has nothing to do with this type of build. Its already there after scaffolding.

Same is true for package.json etc.